.github/workflows/dependabot.yml

```
@@ -12,7 +12,7 @@ jobs:
       pull-requests: write
     steps:
       - uses: actions/checkout@v5
-      - uses: actions/setup-node@v4
+      - uses: actions/setup-node@v5
         with:
           node-version: 18
```

```suggestion
          node-version: lts/*
```

We should probably stop using a deprecated version of NodeJS.